pkg/config/config.go

```diff
@@ -19,6 +19,8 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+const DefaultWebUIAddr = ":8080"
```

Comment:
```suggestion
const DefaultWebUIPort = ":8080"
```

Reply:
err, changed to WebUIListenAddr. it includes also the bind addr, e.g. "127.0.0.1:8080" is valid input